src/scss/foundations/breakpoints.scss

```diff
@@ -0,0 +1,65 @@
+@use "../lib";
+
+$_breakpointBase: 1px;
+$_breakpointXsmall: 576px;
+$_breakpointSmall: 768px;
+$_breakpointMedium: 992px;
+$_breakpointLarge: 1200px;
+$_breakpointXlarge: 1920px;
+
+/**
+ * The breakpoints of Pyxis
+ *
+ * @type Map
+ * @access private
+ */
+$_breakpoints: (
+  base: $_breakpointBase,
```

Comment:
Forse in effetti questo non dovremmo mai definirlo, in quanto non esiste un _punto di rottura_ nella risoluzione minima. 

Invece possiamo censire quella che per noi è la risoluzione minima supportata (`320px` in larghezza?). 
Mi viene in mente che potrebbe tornare utile in certi contesti come i containers e in un breve futuro le _container queries_.

Reply:
Si anche sui `320px` ci pensavo, sono sempre meno usati, ma continuiamo a supportarli. Potremmo effettivamente includere un nuovo breakpoint oppure sostituire totalmente base